src/main/java/net/backupcup/mcde/Config.java

```diff
@@ -294,6 +318,10 @@ public boolean isUsingEnchantingTableAllowed() {
         return allowUsingEnchantingTable;
     }
 
+    public boolean doesFullRerollRemoveSlots() {
+        return fullRerollCanRemoveSlots;
```

Comment:
Might want to consider renaming this `canFullRerollRemoveSlots` to be more in line with common boolean nomenclature